db/re/item_db_equip.yml

```diff
@@ -50445,11 +50442,8 @@ Body:
       NoDrop: true
     Script: |
       bonus2 bSubRace,RC_Demon,3;
-      bonus2 bHPDrainRate,3,15;
-      bonus2 bSPDrainRate,1,7;
-      if (vip_status(VIP_STATUS_ACTIVE)) {
-         bonus2 bSubRace,RC_Demon,2;
-      }
+      bonus2 bHPDrainRate,30,15;
+      bonus2 bSPDrainRate,10,7;
```

Comment:
The item script doesn't match with KRO description
https://www.divine-pride.net/database/item/5584
```
Reduces damage to demon-type enemies by 3%.
Recovers 1 SP when killing an enemy with a melee physical attack or magic attack.
When used in a Gold PC room, the amount of SP recovered is added by 2, and damage reduction to demon-type enemies is added by 2%.
```
For me the item script should be
```
    Script: |
      bonus2 bSubRace,RC_Demon,3;
      bonus bSPGainValue,1;
      bonus bMagicSPGainValue,1;
      if (vip_status(VIP_STATUS_ACTIVE)) {
         bonus bSPGainValue,2;
         bonus bMagicSPGainValue,2;
         bonus2 bSubRace,RC_Demon,2;
      }
```

Reply:
Thanks !